javascript/javascript2/week2/homework app2/main.css

```diff
@@ -0,0 +1,131 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: "Open Sans", sans-serif;
+    background-color: #f9fbfd;
+}
+
+header {
+    width: 100%;
+    height: 90px;
+    display: block;
+    background-color: #f0b2a8;
+}
+
+.navigation {
+    width: 100%;
+    margin: 0 auto;
+}
+
+.logoContainer {
+    height: 100%;
+    display: table;
+    float: left;
+    padding-left: 20px;
+}
+
+.logoContainer h1 {
+    color: white;
+    height: 100%;
+    display: table-cell;
+    vertical-align: middle;
+    font-size: 32px;
+    font-weight: 50;
+}
+
+.logoContainer h1 span {
+    font-weight: 1000;
+}
+
+.navigation {
+    height: 100%;
+    display: table;
+    float: left;
+    padding: 5px 0px 10px 0px;
+}
+
+.navigation li {
+    display: table-cell;
+    justify-content: space-between;
+    vertical-align: middle;
+    height: 100%;
+    color: white;
+    font-size: 16px;
+    padding: 25px 50px 10px 50px;
+}
+
+.searchBar {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: auto;
+    background-color: #e26d5c;
+    padding: 0.5rem;
+    border-radius: 0.5rem;
+    color: white;
+    box-shadow: 0.3rem 0.3rem 0.3rem #f0b2a8;
+}
+
+h2 {
+    padding: 20px 60px 10px 80px;
+}
+
+input {
+    margin: 0 0.5rem 0 0.5rem;
+    width: 100%;
+    border: none;
+    outline: none;
+    background-color: #e26d5c;
+    color: white;
+}
+
+::placeholder {
+    color: #f0b2a8;
+}
+
+footer {
+    background-color: #f0b2a8;
+    color: white;
+    margin-top: 100px;
+    padding: 40px 20px 40px 20px;
+}
+
+.footerNavigation {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+}
+
+.listOfProducts {
+    margin: 20px 100px 20px 50px;
+    font-size: 20px;
+}
+
+@media screen and (max-width: 480px) {
```

Comment:
So nice that you made it also responsive !

Reply:
Thank you so much for reviewing my homework. I`ll change everything what you have mentioned:)